dotCMS/src/main/java/com/dotcms/util/MimeTypeUtils.java

```diff
@@ -46,8 +42,9 @@ public static String getMimeType (final File binary) {
             if( !UtilMethods.isSet(mimeType)){
                 try {
                     mimeType = new TikaUtils().detect(binary);
-                } catch(Exception e) {
```

Comment:
What if these throw a throwable?  I am not sure `Sneeky.sneek` handles throwables. 
```
        final Path path = binary.toPath();
        String mimeType = Sneaky.sneak(() -> Files.probeContentType(path));
```

Reply:
Rats!! they don't!